src/pages/Home/Home.jsx

```diff
@@ -1,12 +1,156 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
 import Header from '../../components/Header/Header';
+import Card from '../../components/Card/Card';
+import { geraHexadecimal } from '../../helpers/utils';
+import { BASE_URL, fetchApi } from '../../api/config';
 
+import albumimg from '../../images/album.png'
 
 import './Home.scss';
 
-const Home = () => (
-  <Header />
-  
-);
+const Home = ({ token }) => {
+
+  const [artists, setArtists] = useState([]) //pesquisar artistas
+  const [albums, setAlbums] = useState([]) //pesquisar álbuns
+  const [severalAlbums, setSeveralAlbums] = useState([]) //página inicial randomica artistas
+  const [severalCategories, setSeveralCategories] = useState([]) //página inicial randomica categorias
+  const [isLoading, setisLoading] = useState(false)
+
+  let navigate = useNavigate() //redirecionar para a home quando o token for inválido
+
+  useEffect(() => {
+    const getAlbums = async () => {
+      const { data } = await fetchApi(`${BASE_URL}/browse/new-releases`, token)
+      setSeveralAlbums([...data.albums.items])
+    }
+
+    if(token) {
+      getAlbums()
+    }
+  }, [token])
+
+  useEffect(() => {
+    const getCategories = async () => {
+      const { data } = await fetchApi(`${BASE_URL}/browse/categories`, token)
+      setSeveralCategories([...data.categories.items])
+    }
+
+    if(token) {
+      getCategories()
+    }
+  }, [token])
```

Comment:
We can use only one useEffect here and declare the functions outside the useEffect.
```suggestion
    const getAlbums = async () => {
      const { data } = await fetchApi(`${BASE_URL}/browse/new-releases`, token)
      setSeveralAlbums([...data.albums.items])
    }       
     const getCategories = async () => {
      const { data } = await fetchApi(`${BASE_URL}/browse/categories`, token)
      setSeveralCategories([...data.categories.items])
    }
    
  useEffect(() => {
   if(token) {
      getAlbums()
      getCategories()

    }
  }, [token])
```